---
sensor_msgs/Image color_image
sensor_msgs/Image depth_image
bool success
string message  # for error description